tests/copilot_helper/test_command_check_cloudformation.py

```diff
@@ -49,7 +49,7 @@ def application_under_test(copilot_directory):
     CliRunner().invoke(make_config)
     CliRunner().invoke(make_addons)
     yield
-    ensure_directory_does_not_exist(copilot_directory)
+    # ensure_directory_does_not_exist(copilot_directory)
```

Comment:
Does this need to be deleted or kept in?

Reply:
Kept in. I commented it out when debugging something so I could see the files...